### ISSUE TITLE
Handle textbox content

### ISSUE THREAD

**Issue by kylegibson-rldatix** [CenterForOpenScience/pydocx#98]
Textbox content isn't included in the output.

https://s3.amazonaws.com/uploads.hipchat.com/6795/58993/xpNA0rNj1K7pZjC/textboxes.doc

The above file should convert to include "Foo" and "Test" in the output, but does not.

**Comment by jlward** [comment]
That link gives access denied. Do you still have it somewhere?

**Comment by kylegibson** [comment]
Yes, I do, although I am now unsure of how we should attach/share these files.
